api/controllers/WebActionsController.js

```diff
@@ -127,6 +128,77 @@ module.exports = {
     }
   },
 
+  splash: function(req, res) {
+    let { EMAIL, FNAME, PHONE, group } = req.body;
+    let memberHash = crypto
+      .createHash('md5')
+      .update(EMAIL.toLowerCase())
+      .digest('hex');
+    let redirectUrl = '/confirmation';
+
+    const mailchimpUpdateRequest = {
+      method: 'PATCH',
+      uri: `${sails.config.globals.mailchimpApiUrl}/lists/${sails.config
+        .globals.mailchimpListId}/members/${memberHash}`,
+      json: true,
+      auth: {
+        user: sails.config.globals.mailchimpApiAuthUser,
+        pass: sails.config.globals.mailchimpApiAuthPass,
+      },
+      body: {
+        status: 'subscribed',
+        // Subscribe users to Shine Splash App Group
+        interests: { e8db15c44a: true },
+      },
+    };
+
+    const mailchimpSubscribeRequest = {
+      method: 'POST',
+      uri: `${sails.config.globals.mailchimpApiUrl}/lists/${sails.config.globals
+        .mailchimpListId}/members`,
+      json: true,
+      auth: {
+        user: sails.config.globals.mailchimpApiAuthUser,
+        pass: sails.config.globals.mailchimpApiAuthPass,
+      },
+      body: {
+        email_address: EMAIL,
+        status: 'subscribed',
+        merge_fields: {
+          FNAME: FNAME,
+          PHONE: PHONE,
+        },
+        interests: { e8db15c44a: true }, // Subscribe users to Shine Splash App Group
+      },
+    };
+
+    request
+      .patchAsync(mailchimpUpdateRequest)
+      .then(response => {
+        if (!response || !response.body) {
+          sails.log.error(
```

Comment:
Wondering if there's no response returned, if it would automatically go into the error catch block? Idk? 

But typically in this and the `404` case we might want to throw the error to the catch block anyhow and then have the `sail.logging` happen there. (ie. `throw new Error('Invalid response received from MailChimp subscribe`);  See example in `join`, line 260 I think.